tests/privacy_tests/packet_tests/test_packet_port.py

```diff
@@ -0,0 +1,132 @@
+import unittest
+import unittest.mock as um
+from pathlib import Path
+from scapy.all import rdpcap
+from src.privacy_analysis.packet_analysis.packet_privacy_port import PacketPrivacyPort
+
+root_path_test_data = "../../test_data/"
+
+
+class TestIPSignature(unittest.TestCase):
+    def setUp(self) -> None:
+        self.packet_privacy_port = PacketPrivacyPort()
+
+    @um.patch("src.dashboard.alerts.alert.Alert.alert")
```

Comment:
This is something that I just noticed when it was written out we should rename the function under the class send instead of alert. I can do this in my sqlite PR